components/tabcordion/src/Tabcordion.js

```diff
@@ -1,9 +1,10 @@
 import React, { Children, useEffect, useRef, useState } from 'react';
 import PropTypes from 'prop-types';
+import { useBrand, merge } from '@westpac/core';
 import { useContainerQuery } from '@westpac/hooks';
-
-import { TabItem, TabRow } from './styled';
+import { TabItem, TabRow, AccordionLabel, Panel } from './styled';
 import { Tab } from './Tab';
+import pkg from '../package.json';
 
 let instanceId = 0;
```

Comment:
We need to make sure this works when the component is run multiple times on the same page even in different react wrappers (multiple react apps)

Reply:
How do we want to handle unique id's for components? You said before you didnt like generating a random unique id (was using shortid package in other components to do this.)

Reply:
We need to add the unique ID to the `window` object directly.